server/routes/requests.js

```diff
@@ -322,4 +323,53 @@ router.get('/getConfirmed/:id/:timeOffset', passport.authenticate('jwt', {sessio
     }
 })
 
+router.patch('/actionItems/:id/:timeOffset', passport.authenticate('jwt', {session: false}), async (req, res, next) => {
+    let alumniId = req.params.id;
+    let timeOffset = req.params.timeOffset;
+    let requestId = req.body.requestId;
+    let actionItems = req.body.actionItems;
+    let conditions = ['Awaiting Confirmation', 'Confirmed', 'Completed']
+    let requests = []
+    try {
+        let request = await requestSchema.findById(requestId);
+        request.actionItems = actionItems;
+        await request.save()
+        for (let status of conditions) {
```

Comment:
We might not need to send back requests in the response and something like `res.status(200).send({'message' : "Added action items"});`  might be sufficient

Reply:
So we can remove the code where we're fetching the all the requests for the alumni